lux/guides/agents.livemd

```diff
@@ -3,7 +3,7 @@
 ```elixir
 Mix.install(
   [
-    {:lux, "~> 0.4.0"},
+    {:lux, "~> 0.5.0"},
```

Comment:
How about use `>=` operator? There is no need to modify this even if a new minor version comes out later.

```suggestion
    {:lux, ">= 0.5.0"},
```

Reply:
Good point! I've changed it in https://github.com/Spectral-Finance/lux/pull/142/commits/57016e7bb5e9f0eec1430a20aa35735f6274d135